torch/_torch_docs.py

```diff
@@ -3502,6 +3502,36 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.ge`.
 """)
 
+add_docstr(torch.gradient,
+           r"""
+gradient(input, *, spacing=None, dim=None, edge_order=1) -> List of Tensors
+
+This function is analogous to NumPy's gradient function.
+
+Args:
+    input (Tensor): Values of the function that we estimate gradient.
+
+Keyword args:
+    spacing (scalar, list of scalar, list of Tensor, optional): represents either positions of
```

Comment:
we can work on this language a little offline

Reply:
sure, looking forward to it :)